R/ReportCard.R

```diff
@@ -31,12 +35,14 @@ ReportCard <- R6::R6Class( # nolint: object_name_linter.
     },
     #' @description Appends a plot to this `ReportCard`.
     #'
-    #' @param plot the appended plot
-    #' @param dim `integer vector` width and height in pixels.
-    #' @return invisibly self
+    #' @param plot (`ggplot` or `grob` or `trellis`) plot object.
+    #' @param dim (`integer(2)`) width and height in pixels.
```

Comment:
Doubles may also be accepted, please verify and if so, change to "`numeric`".

Reply:
yes, it does accept. changing